includes/definitions/pbi.yaml

```diff
@@ -0,0 +1,10 @@
+os: pbi
+text: 'Linux'
```

Comment:
text should describe the OS / Product line.

Reply:
@laf There is no MIB for that device, I can't find any, only that OIDs, this device is running linux, there is a tab with version and that linux os version 1b is not listed in snmpwalk.

![snimka obrazovky 2018-08-25 o 16 42 51](https://user-images.githubusercontent.com/36922215/44619392-ff2c2500-a885-11e8-8282-3447cc9fd934.png)

What do you think, when are you talking about correct name ? 

I am not programmer, so when is code bad, please help me with this. I am just trying to add some devices, which we have on our network and will be really awesome, when it will be possible to monitor them. 

Thank you.

Reply:
You are right, I changed os to IRD